README.md

```diff
@@ -144,6 +144,14 @@ To reboot after successfully update, run:
 
 You can use it to automate updates.
 
+#### `WANT_PI4`
+
+If you want the tool to also download the Raspberry Pi 4 firmware, run:
+
+    sudo WANT_PI4=1 rpi-update
+
+By default, Pi 4 firmware is not downloaded on system running with a lower pi version.
```

Comment:
This isn't quite right.

> By default, Pi 4 firmware is not downloaded on system that does not currently have a Pi 4 kernel

is more correct. (Specifically it checks for /boot/kernel7l.img and assumes if that exists it should update all Pi 4 files)